project.clj

```diff
@@ -1,4 +1,4 @@
-(defproject metosin/spec-tools "0.1.0"
+(defproject metosin/spec-tools "0.1.1"
```

Comment:
Could you remove this? I'd prefer to not have version bumps in PRs, and it should be to a `-SNAPSHOT` release first anyway.